Components/Badge/badge.css

```diff
@@ -0,0 +1,91 @@
+
+.badge-container{
+    display: flex;
+    position: relative;
+    width: fit-content;
+    font-size: 2rem;
+}
+
+.status-badge{
+    border-radius: 50%;
+    border: 2px solid #fff;
```

Comment:
declare colors as variable & use it here

Reply:
Sure, will do that thanks!